src/intra_edge.rs

```diff
@@ -18,18 +18,23 @@ bitflags! {
 }
 
 impl EdgeFlags {
-    const LEFT_HAS_BOTTOM: Self = Self::union_all([
+    pub const LEFT_HAS_BOTTOM: Self = Self::union_all([
         Self::I444_LEFT_HAS_BOTTOM,
         Self::I422_LEFT_HAS_BOTTOM,
         Self::I420_LEFT_HAS_BOTTOM,
     ]);
 
-    const TOP_HAS_RIGHT: Self = Self::union_all([
+    pub const TOP_HAS_RIGHT: Self = Self::union_all([
         Self::I444_TOP_HAS_RIGHT,
         Self::I422_TOP_HAS_RIGHT,
         Self::I420_TOP_HAS_RIGHT,
     ]);
 
+    pub const EDGE_ALL_TR_AND_BL: Self =
+        Self::union_all([Self::LEFT_HAS_BOTTOM, Self::TOP_HAS_RIGHT]);
+
+    pub const EDGE_NONE: Self = Self::empty();
```

Comment:
I think we can drop the `EDGE_` prefix from these constants. They'll always occur qualified as `EdgeFlags::*` so the edge bit is just repeated.